walkthroughs/tls-with-acm/mesh/mesh.yaml

```diff
@@ -40,16 +40,28 @@ Resources:
                   ACM:
                     CertificateAuthorityArns:
                       - !Ref CertificateAuthorityArn
-        Backends:
-          - VirtualService:
-              VirtualServiceName: !Sub "colorteller.${ServicesDomain}"
         Listeners:
           - PortMapping:
               Port: 80
               Protocol: http
-        ServiceDiscovery:
-          DNS:
-            Hostname: !Sub "colorgateway.${ServicesDomain}"
+
+  ColorGatewayRoute:
+    DependsOn:
+      - ColorGatewayVirtualGateway
+      - ColorTellerVirtualService
+    Type: AWS::AppMesh::GatewayRoute
+    Properties: 
+      GatewayRouteName: ColorGatewayRoute
+      MeshName: !GetAtt Mesh.MeshName
+      Spec: 
+        HttpRoute:
+          Action: 
+            Target:
+              VirtualService:
+                VirtualServiceName: !Sub "colorteller.${ServicesDomain}"
+          Match: 
+            Prefix: /
+      VirtualGatewayName: ColorGateway
```

Comment:
minor: Instead of hardcoding names, you can just !GetAtt the names of the resources. e.g. `!GetAtt ColorGatewayVirtualGateway.VirtualGatewayName`. That will also set up dependencies automatically so you don't need the manual DependsOn.